src/components/views/EditMilestone.jsx

```diff
@@ -86,6 +100,7 @@ class EditMilestone extends Component {
   componentDidMount() {
     checkForeignNetwork(this.props.isForeignNetwork)
       .then(() => this.checkUser())
+      .then(() => this.props.getEthConversion(this.state.date, this.state.milestone.token.symbol))
```

Comment:
You are checking conversion before the symbol is set. This could potentially lead to incorrect exchange rate.

Reply:
OK I can check that but I don't think adding it below and leaving it here makes sense.

Reply:
Sure! Please do!